pandas/tests/test_base.py

```diff
@@ -263,77 +263,76 @@ def setup_method(self, method):
-
-                # noinspection PyComparisonWithNone
-                result = o == None  # noqa
-                assert not result.iat[0]
-                assert not result.iat[1]
-
-                # noinspection PyComparisonWithNone
-                result = o != None  # noqa
-                assert result.iat[0]
-                assert result.iat[1]
-
-                result = None == o  # noqa
-                assert not result.iat[0]
-                assert not result.iat[1]
-
-                result = None != o  # noqa
-                assert result.iat[0]
-                assert result.iat[1]
-
-                if (is_datetime64_dtype(o) or is_datetime64tz_dtype(o)):
-                    # Following DatetimeIndex (and Timestamp) convention,
-                    # inequality comparisons with Series[datetime64] raise
-                    with pytest.raises(TypeError):
-                        None > o
-                    with pytest.raises(TypeError):
-                        o > None
-                else:
-                    result = None > o
-                    assert not result.iat[0]
-                    assert not result.iat[1]
+            result = s < None
+            assert not result.iat[0]
+            assert not result.iat[1]
 
-                    result = o < None
-                    assert not result.iat[0]
-                    assert not result.iat[1]
+    def test_ndarray_compat_properties(self, all_indices_and_series_fixture):
 
-    def test_ndarray_compat_properties(self):
+        o = all_indices_and_series_fixture
+        # Check that we work.
+        for p in ['shape', 'dtype', 'T', 'nbytes']:
+            assert getattr(o, p, None) is not None
 
-        for o in self.objs:
-            # Check that we work.
-            for p in ['shape', 'dtype', 'T', 'nbytes']:
+        # deprecated properties
+        for p in ['flags', 'strides', 'itemsize']:
+            with tm.assert_produces_warning(FutureWarning):
                 assert getattr(o, p, None) is not None
 
-            # deprecated properties
-            for p in ['flags', 'strides', 'itemsize']:
-                with tm.assert_produces_warning(FutureWarning):
-                    assert getattr(o, p, None) is not None
+        with tm.assert_produces_warning(FutureWarning):
+            assert hasattr(o, 'base')
 
+        # If we have a datetime-like dtype then needs a view to work
+        # but the user is responsible for that
+        try:
             with tm.assert_produces_warning(FutureWarning):
-                assert hasattr(o, 'base')
-
-            # If we have a datetime-like dtype then needs a view to work
-            # but the user is responsible for that
-            try:
-                with tm.assert_produces_warning(FutureWarning):
-                    assert o.data is not None
-            except ValueError:
-                pass
+                assert o.data is not None
+        except ValueError:
```

Comment:
this is weird in master, what is causing the ValueError here?

Reply:
I think it is been there for a long time. shall I remove the try-except block ?

Reply:
yes see if you can